config/config_test.go

```diff
@@ -48,7 +48,7 @@ route:
    group_wait: 30s
 `
 	conf := &Config{}
-	err := yaml.Unmarshal([]byte(in), conf)
+	err := yaml.UnmarshalStrict([]byte(in), conf)
```

Comment:
It would be good to use the same pattern as in the other tests:

```git
-       conf := &Config{}
-       err := yaml.Unmarshal([]byte(in), conf)
+       _, err := Load(in)
```

Reply:
I changed it. It was doing the same job predefined in `Load`, so there is no need for it. Thanks!